documentation/onboard-dpg-in-sdkautomation/.net/README.md

```diff
@@ -1,22 +1,13 @@
-# Add Autorest Configuration of .Net SDK
+# Add TypeSpec Configuration of .Net SDK
 
-## Parameter Description
+Please copy the following configuration into `tspconfig.yaml` in your spec folder and replace `package-dir` and `namespace` to your own values.
 
-- `<ServiceName>`: The RP name, which is usually same as folder name in swagger.
-- `<NameSpace>`: Python package name.
-- `<Title>`: SDK Client Name. It's optional if there is a title defined in spec readme.md file.
-
-## Autorest Configuration
-Please copy the following configuration into spec PR comment.
-~~~
-# azure-sdk-for-net-track2
 ``` yaml
-title: <Title>
-output-folder: sdk/<ServiceName>/<NameSpace>
-require:
- - specification/<RPName>/data-plane/readme.md
+  "@azure-tools/typespec-csharp":
+    package-dir: "Azure.AI.Vision.Face"
+    flavor: azure
+    namespace: "{package-dir}"
 ```
-~~~
-- `title`: If it's already defined in spec readme.md, you don't need to define it here again. (By default, there is no `title` defined in spec readme.md in multi client scenario.)
-- `output-folder`: The relative path of destination to generate SDK.
-- `require`: The item of the value is the relative path of spec readme.md file.
+- `package-dir`: Name of your package directory.
+- `flavor`: Always `azure` for Azure SDK.
+- `namespace`: Namespace of your generated SDK.
```

Comment:
Can we add some .Net configuration constrains here? such as `namespace` should be the same as `package-dir`, and the name conversion of the namespace?